### PR TITLE
Update app tute

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

**Comment by katejeffreys** [comment]
Addresses https://github.com/Polymer/docs/issues/2018

File: app/2.0/start/toolbox/create-a-page.md [hunk 1]
```diff
@@ -35,15 +36,15 @@ your new view.
...
           static get is() { return 'my-new-view'; }
         }
+        //Now, register your new custom element so the browser can use it
         customElements.define(MyNewView.is, MyNewView);
       </script>
     </dom-module>
-
     ```
 
 For now your element is very basic, and just has a `<h1>` that says "New View",
```
Comment (FredKSchott) [inline app/2.0/start/toolbox/create-a-page.md]: "New view" vs. "New View", the casing is off here.

File: app/2.0/start/toolbox/create-a-page.md [hunk 2]
```diff
@@ -168,10 +169,11 @@ just need to add a menu item in the left-hand drawer so that users can navigate
     ...
     ```
 
-Your new page is now ready! Open your web browser and view it at
-[http://localhost:8080/new-view](http://localhost:8080/new-view).
+Your new page is now ready! Serve your app with `polymer serve`, then open the resulting URL in your browser.
+
+![Output from polymer serve command](/images/2.0/toolbox/polymer-serve-output.png)
```
Comment (FredKSchott) [inline app/2.0/start/toolbox/create-a-page.md]: these images don't show up in preview. Could be a github issue, or the `app/` path prefix is missing

File: app/2.0/start/toolbox/deploy.md [hunk 3]
```diff
@@ -143,10 +162,33 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
...
+      },
+      "hosting": {
+        "public": "build/default",
+        "rewrites": [
+          {
+            "source": "!/__/**",
+            "destination": "/index.html"
+          },
+          {
+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)",
```
Comment (FredKSchott) [inline app/2.0/start/toolbox/deploy.md]: is there a more general `*.*` that we could suggest to users here to check for all URLs that don't include file extensions?

File: app/2.0/start/toolbox/create-a-page.md (hunk 2 above)
Comment (katejeffreys) [inline app/2.0/start/toolbox/create-a-page.md]: They are fine when I build and test

File: app/2.0/start/toolbox/deploy.md (hunk 3 above)
Comment (katejeffreys) [inline app/2.0/start/toolbox/deploy.md]: Could you clarify what you mean? 

Do you mean change 

`{+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)", `

to 

`{+            "source": "**/!(*.*)",`

?

I can definitely do that if that's what you mean.

**Comment by katejeffreys** [comment]
Thanks Arthur and Fred

I've addressed a couple of points and requested clarification of another.

Kate

File: app/2.0/start/toolbox/deploy.md (hunk 3 above)
Comment (jsilvermist) [inline app/2.0/start/toolbox/deploy.md]: I'm not sure about an alternative, but it makes more sense to me to use something like this (not sure if this is valid though):
```js
"source": "**/!*.(html|css|js|json|svg|png|jpg|jpeg)",
```

**Comment by arthurevans** [comment]
Tested locally, images look good.

Since the only question at this point is whether we should change that one line of the deploy config, I'm going to merge this and we can take that up in a  separate PR, if necessary.

**Pull request merged**